patches/@mapbox+tiny-sdf+2.0.5.patch

```diff
@@ -0,0 +1,183 @@
+diff --git a/node_modules/@mapbox/tiny-sdf/index.cjs b/node_modules/@mapbox/tiny-sdf/index.cjs
```

Comment:
The patches are simply copying the `dist/*.js` files to `dist/*.cjs`. d3 is quite clear about their move to `type: module` and no longer supporting the commonjs syntax. We are unable to stay on older versions of the packages because of security warnings. Our repo cannot "just work" with esm-only packages unless we add `type:module` to all our package.json also.

Reply:
But why are they listed as patches in this diff? Are those files committed by mistake? Can copy them in any way that doesn't involve our repo. 

I understand the d3 situation they made the change to `type: module` a long time ago. Do we really need to be on v7 if we plan to stay non-esm?

Any chance we are ready to go "esm" for v9? I would vote for it, as it is increasingly a pain to support non-esm setups.

Reply:
> do we really need to be

Yes, e.g. d3-color has a security vulnerability that is only patched in 3.1.0. 
I can copy those files with a script if that makes you feel better.